### DISCUSSION TITLE
Decouple admin-ui dependency from base keystone packages

### DISCUSSION THREAD

**Discussion by gautamsi** [keystonejs/keystone#5650] [Feature requests]
This is related to #5649.

Another blocker for further extensibility is that base keystone packages depends on `admin-ui` which hinders the customisation of admin-ui by forking it.

I was trying to see if I could refactor some of the part myself and customise admin-ui, but realised that it is not possible due to dependency as well as use of `admin-ui/system` in cli.

I think we should also separate cli from the base keystone package

**Comment by bladey** [comment]
Thanks for providing this suggestion @gautamsi.

**Comment by gautamsi** [comment]
seems like we are doing opposite of this so may be we could archive this thread
